src/vaults/restaking/UltraLRT.sol

```diff
@@ -153,8 +155,9 @@ contract UltraLRT is
     }
 
     /// @notice Each wei of `asset` at genesis is worth 10 ** (initialShareDecimals) shares.
+    /// @dev zero decimals means 1:1 ratio
     function _initialShareDecimals() internal pure virtual returns (uint8) {
-        return 8;
+        return 0;
```

Comment:
IMPORTANT: can you just instead hardcode decimal to 0 and remove these altogether? we dont need to do this extramath for decimal